inc_internal/internal_model.h

```diff
@@ -148,7 +148,7 @@ DECLARE_MODEL(ziti_edge_router, ZITI_EDGE_ROUTER_MODEL)
 
 DECLARE_MODEL(ziti_net_session, ZITI_NET_SESSION_MODEL)
 
-DECLARE_MODEL(ziti_session, ZITI_SESSION_MODEL)
+DECLARE_MODEL(ziti_session, ZITI_API_SESSION_MODEL)
```

Comment:
any reason to rename the model but not the type?

Reply:
Rename completed.